### DISCUSSION TITLE
Plot: How to query currently hidden items

### DISCUSSION THREAD

**Discussion by metasim** [emilk/egui#4492] [Q&A] [answered]
I'm trying to determine if a plot item is visible or not so that the state of another control can be affected by it. I see this method on `LegendWidget`...

https://github.com/emilk/egui/blob/44d65f41ac297847ceedc452c6a52299fe17f4a9/crates/egui_plot/src/legend.rs#L219-L226

...but I can't figure out how to get access to the `LegendWidget` from the `Response` (or other pathway). Is this currently possible? 

Note, I've only been using `egui` for a few weeks, so apologies if this is a beginner question.

**Comment by YgorSouza** [comment, marked as answer]
I think the way to do that currently is to call [PlotMemory::load](https://docs.rs/egui_plot/0.27.2/egui_plot/struct.PlotMemory.html#method.load), using the same [id](https://docs.rs/egui_plot/0.27.2/egui_plot/struct.Plot.html#method.id) you used for the plot itself.
